web/styles/right_sidebar.css

```diff
@@ -161,6 +161,13 @@ $user_status_emoji_width: 24px;
         }
     }
 
+    .user_sidebar_entry.with_avatar .user_circle {
+        display: inline-block;
+        position: absolute;
+        top: 32px;
+        left: 32px;
```

Comment:
There are some magic numbers here, these are just "what looked reasonable" -- I'm not sure what we actually want in terms of placement. I'm happy to make it a variable if that's preferred.

Reply:
That's looking a lot like 2em given our 16px default font size? We will want it to scale if font size is changed.